main/2024/CG-07-16.md

```diff
@@ -15,6 +15,9 @@ No registration is required for VC meetings. The meeting is open to CG members o
 1. Opening
 1. Proposals and discussions
    1. FP16 proposal: next steps (Ilya Rezvov, 45 minutes)
+   1. Phase 4 vote for
```

Comment:
Let's maybe put this first; then if it takes less than 10 minutes (or up to 15, assuming nothing else gets added to the agenda before then), FP16 can just take the remaining time.

Reply:
Done